scripts/2.TCGA-process.py

```diff
@@ -163,12 +163,21 @@
 clinmat_df = clinmat_df.query('Redaction != "Redacted"').drop("Redaction", axis=1)
 
 # Keep only these sample types
-# filters duplicate samples per patient
 sample_types = {
     'Primary Solid Tumor',
+    'Recurrent Solid Tumor',
     'Primary Blood Derived Cancer - Peripheral Blood',
+    'Metastatic',
 }
-clinmat_df.query("sample_type in @sample_types", inplace=True)
+
+clinmat_df = (
+    clinmat_df
```

Comment:
is this already sorted elsewhere? It is not explicitly sorted in this block

Reply:
Yep sorted in line 125 above